src/styles/blocks/card.scss

```
@@ -0,0 +1,96 @@
+.card {
+  width: $card-width;
+  padding: 16px;
+  padding-top: 32px;
```

You can combine these paddings
```suggestion
  padding: 32px 16px 16px;
```